app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -197,7 +197,11 @@ def curation_concern_from_search_results
 
     def search_result_document(search_params)
       _, document_list = search_results(search_params)
-      raise CanCan::AccessDenied.new(nil, :show) if document_list.empty?
+      if (_curation_concern_type.find(params[:id]).suppressed?)
```

Comment:
This is going to be too slow.  We want to query this from Solr, not from Fedora.